viscy/data/hcs.py

```diff
@@ -26,6 +27,11 @@
 
 from viscy.data.typing import ChannelMap, DictTransform, HCSStackIndex, NormMeta, Sample
 
+warnings.filterwarnings(
+    "ignore",
+    category=UserWarning,
+    message="To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).",
+)
```

Comment:
If there is a warning, it should be fixed instead of suppressed. Also I believe I already fixed this.